src/app/routes.php

```diff
@@ -15,6 +15,7 @@
     "/logout/"              => "/static_page/logout/",
     "/register/"            => "/static_page/register/",
     "/scores/codecraft/"    => "/static_page/codecraft/",
+    "/wdw/"                 => "/static_page/wdw/",
```

Comment:
Oh this because in web development workshop's poster, Jeevan didn't write the URL for it, so we got it edited there itself, and because "talks-and-workshops/web-development" was too big, so we wrote this.